src/pages/api/logs/[id].js

```diff
@@ -210,8 +248,7 @@ export default async function handler(req, res) {
         });
       }
       // succcess for user/admin
-      data.resolver = session.user._id;
```

Comment:
we still need the `resolver` here
![image](https://github.com/GTBitsOfGood/canine-assistants/assets/45372449/3a08b715-63d0-4a65-888a-b659abc18dad)